src/cmd-koji-upload

```diff
@@ -924,6 +955,8 @@ Environment variables are supported:
     search_cmd = sub_commands.add_parser(
         "search", help="Search for a build", parents=[parent_parser], add_help=False)
 
+    ensure_tag = sub_commands.add_parser(
```

Comment:
```suggestion
    ensure_cmd = sub_commands.add_parser(
```

Reply:
You are too fast. I was fixing it when I saw your comment =)